include/exadg/operators/mass_kernel.h

```diff
@@ -19,8 +19,8 @@
  *  ______________________________________________________________________
  */
 
-#ifndef INCLUDE_OPERATORS_MASS_MATRIX_OPERATOR_H_
-#define INCLUDE_OPERATORS_MASS_MATRIX_OPERATOR_H_
+#ifndef INCLUDE_OPERATORS_MASS_KERNEL_H_
+#define INCLUDE_OPERATORS_MASS_KERNEL_H_
```

Comment:
I think it is time to switch to `#pragma once`.

Reply:
feels like something we should discuss before someone is changing **all** the files :laughing:
Is the argument that it is not part of the standard still true actually? I read [this discussion](https://stackoverflow.com/questions/787533/is-pragma-once-a-safe-include-guard), which is from 2009 ...
Any other opinions on this @nfehn @kronbichler ?